src/Traces.Web/ViewModels/TracesBaseViewModel.cs

```diff
@@ -193,7 +193,11 @@ protected void AddTraceToDictionary(TraceItemModel trace)
         {
             if (SortedGroupedTracesDictionary.ContainsKey(trace.DueDate))
             {
-                SortedGroupedTracesDictionary[trace.DueDate].Add(trace);
+                var list = SortedGroupedTracesDictionary[trace.DueDate];
```

Comment:
```suggestion
                var existentTraces = SortedGroupedTracesDictionary[trace.DueDate];
```
Maybe a more descriptive name. That is just a suggestion.